topics/go/algorithms/numbers/reverse/reverse.go

```diff
@@ -0,0 +1,24 @@
+package numbers
+
+// Reverse takes in a integer and returns a the integer in reverse order.
+func Reverse(num int) int {
+
+	// Set result to zero.
+	var result int
+
+	// Loop until num is not zero.
+	for num != 0 {
+
+		// Get the last digit from num.
+		// Example: 125 % 10 = 5.
+		last := num % 10
+
+		// Move result one place to the left and add last.
```

Comment:
// Multiple the current result by 10 to shit the digits in
// the current result to the left.
result = result*10

// Add the digit we took from the end of num to the result.
result += last

Break these two lines into different steps and explain it